json-proxy-core/build.gradle

```diff
@@ -0,0 +1,29 @@
+description = "A simple server-side JSON proxy library"
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+dependencies {
+    compile 'org.codehaus.groovy:groovy-all:2.4.4'
+    compile 'org.apache.commons:commons-lang3:3.3.2'
+    compile 'commons-codec:commons-codec:1.1'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.2.0'
+    compile 'com.google.guava:guava:18.0'
+    compile 'org.springframework:spring-core:4.1.5.RELEASE'
+    compile 'org.springframework:spring-web:4.1.5.RELEASE'
+    compile 'org.springframework:spring-webmvc:4.1.5.RELEASE'
+    compile 'org.slf4j:jcl-over-slf4j:1.7.5'
+    compile 'org.slf4j:slf4j-api:1.7.5'
+    runtime 'ch.qos.logback:logback-classic:1.0.12'
+    runtime 'org.slf4j:jul-to-slf4j:1.7.5'
+    runtime 'org.slf4j:log4j-over-slf4j:1.7.5'
+    testCompile 'junit:junit:4.11'
+    testCompile 'org.mockito:mockito-core:1.9.5'
+    testCompile 'org.springframework:spring-test:4.1.5.RELEASE'
+    compile('javax.servlet:javax.servlet-api:3.1.0') { // TODO: was 3.0.1 intentional?
+       /* This dependency was originally in the Maven provided scope, but the project was not of type war.
+       This behavior is not yet supported by Gradle, so this dependency has been converted to a compile dependency.
+       Please review and delete this closure when resolved. */
```

Comment:
It's a weird and uncharacteristic blind spot that gradle [does not have a build in 'provided' scope](https://issues.gradle.org/browse/GRADLE-784).  Apparently everyone just defines it themselves until gradle comes around, so I updated with that change.

FYI / for reference, 'provided' means a dependency _is_ needed for compilation, but _is not_ meant to be packaged into the final artifact, because that artifact is expected to wind up in a container (e.g. tomcat, jetty) that already has the dependency.  Compile often works just as well, but can lead to weird conflicts at runtime.